app/models/legal_aid_application.rb

```diff
@@ -18,6 +18,8 @@ class LegalAidApplication < ApplicationRecord
   has_one :statement_of_case
   has_many :legal_aid_application_restrictions
   has_many :restrictions, through: :legal_aid_application_restrictions
+  has_many :legal_aid_application_transaction_types
```

Comment:
is this line needed if we never call it in the code?

Reply:
Yes - without it the next line won't work.